megcruzen/browserify_productOrdering
# Products now display their respective reviews

# Description
Products now display their respective reviews.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout fetch-correct-reviews`
3. Run grunt
4. Verify that product #1 shows two reviews and product #2 shows one review. 

# Checklist:
